dev/com.ibm.websphere.appserver.features/visibility/private/io.openliberty.jakarta.concurrency-3.0.feature

```diff
@@ -5,6 +5,6 @@ singleton=true
 #TODO remove toleration of eeCompatible-9.0 once other EE 10 features are usable in beta form
 -features=com.ibm.websphere.appserver.eeCompatible-10.0; ibm.tolerates:="9.0"
 -bundles=io.openliberty.jakarta.concurrency.3.0; location:="dev/api/spec/,lib/"; mavenCoordinates="io.openliberty.jakarta.enterprise.concurrent:jakarta.enterprise.concurrent-api:3.0.0.20220120"
```

Comment:
perhaps doesn't matter so much for a beta but these maven co-ords are wrong.
```suggestion
-bundles=io.openliberty.jakarta.concurrency.3.0; location:="dev/api/spec/,lib/"; mavenCoordinates="io.openliberty.jakarta.enterprise.concurrent:jakarta.enterprise.concurrent-api:3.0.0.20211206"
```

Reply:
I don't see how 20220120 is wrong and the older 20211206 would be correct.  3.0.0.20220120 is how I versioned Liberty's [copy](http://public.dhe.ibm.com/ibmdl/export/pub/software/olrepo/io/openliberty/jakarta/enterprise/concurrent/jakarta.enterprise.concurrent-api/3.0.0.20220120/) of the Concurrency spec API jar that is the latest candidate for release of that specification which was built on January 20 (matching the version number).  20211206 is an older development built snapshot driver.  As you point, by the time this is GA (and probably well before that point), we will be able to switch this to the real location on Maven of the official final release of the spec API jar.

Reply:
Yeah, sorry, I looked at the wrong file, my fault